lib/vanagon/project.rb

```diff
@@ -101,6 +101,22 @@ def get_replaces
       replaces.flatten.uniq
     end
 
+    # Grabs a specific service based on which name is passed in
+    # note that if the name is wrong or there was no
+    # @component.install_service call in the component, this
+    # will return nil
+    #
+    # @param [string] name of service to grab
+    # @return [@component.service obj] specific service
+    def get_service(name)
+      @components.each do |component|
+        if component.name == name
```

Comment:
What happens if this is run on a component that doesn't have a name? I think we'll get a error for an undefined method. That also might be fine, since I think every other platform adds name to the service object.